measure_cocoapod_size.py

```diff
@@ -165,6 +179,21 @@ def Main():
       nargs='+',
       required=False,
       help='The set of spec_repos')
+  parser.add_argument(
+      '--cocoapods_source_config',
```

Comment:
Do you think we should do any validation on the command/config? 
I'm not sure if validation or documentation is best but here are two edge cases I thought of:
1. Running `/measure_cocoapod_size.py --cocoapods AFNetworking:3.2.0  --cocoapods_source_config config.json` will ignore `--cocoapods_source_config` because a version has been specified in the first argument.
2. The config json could contain a typo like "giit" or and that would break things later on pod install.
WDYT?

Reply:
1. I think if we have version specified, then we might skip the config, since if we have path or git/branch then we don't need version right? Correct me if I am wrong. If so, I will add it to `--help` and print the log if 'cocoapods_source_config' is overridden.
2. Yep we can validate the JSON by making sure the key words are in {'git','path','branch','commit','tag'}

Reply:
SGTM 😃